SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialWorkerConnection.cpp

```diff
@@ -56,6 +56,8 @@ struct ConfigureConnection
 		Params.network.modular_kcp.upstream_heartbeat = &HeartbeatParams;
 #endif
 
+		Params.network.modular_kcp.security_type = GetDefault<USpatialGDKSettings>()->bUseSecureConnection ? WORKER_NETWORK_SECURITY_TYPE_TLS : WORKER_NETWORK_SECURITY_TYPE_INSECURE;
```

Comment:
We also need to disable it for the Modular TCP types as well

Reply:
Truth. Added.